extensions/wikia/AdEngine/js/AdInfoTracker.js

```diff
@@ -0,0 +1,92 @@
+/*global define*/
+define('ext.wikia.adEngine.adInfoTracker',  [
+	'ext.wikia.adEngine.adTracker',
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.lookup.services',
+	'ext.wikia.aRecoveryEngine.recovery.helper',
+	'wikia.log',
+	'wikia.window'
+], function (adTracker, adContext, lookupServices, recoveryHelper, log, window) {
+	'use strict';
+
+	var	logGroup = 'ext.wikia.adEngine.adInfoTracker';
+
+	window.adEnginePvUID = window.adEnginePvUID || generateUUID();
+
+	function prepareData(slot, status) {
+		if (!slot.container.firstChild || !slot.container.firstChild.dataset.gptPageParams) {
+			return;
+		}
+		log(['prepareData', slot, status], 'debug', logGroup);
+		var data,
+			slotFirstChildData = slot.container.firstChild.dataset,
+			pageParams = JSON.parse(slotFirstChildData.gptPageParams),
+			slotParams = JSON.parse(slotFirstChildData.gptSlotParams),
+			slotPrices = lookupServices.getSlotPrices(slot.name),
+			slotSize = JSON.parse(slotFirstChildData.gptCreativeSize);
+
+		data = {
```

Comment:
Is it possible that this URL will be too long? From what I see it shouldn't, but I'm asking just to make sure